performance_manager/lib/gtfs_static_table.py

```diff
@@ -0,0 +1,317 @@
+from typing import List, Optional
+
+import logging
+import pandas
+import numpy
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from .s3_utils import read_parquet, file_list_from_s3
+from .postgres_utils import MetadataLog, get_unprocessed_files
+from .postgres_schema import (
+    StaticFeedInfo,
+    StaticTrips,
+    StaticRoutes,
+    StaticStops,
+    StaticStopTimes,
+    StaticCalendar,
+)
+from .gtfs_utils import start_time_to_seconds
+
+
+def get_static_parquet_paths(table_type: str, feed_info_path: str) -> List[str]:
+    """
+    get static table parquet files from FEED_INFO path
+    """
+    springboard_bucket = "mbta-ctd-dataplatform-dev-springboard"
+    static_prefix = feed_info_path.replace("FEED_INFO", table_type)
+    static_prefix = static_prefix.replace(f"{springboard_bucket}/", "")
+    return list(
+        uri for uri in file_list_from_s3(springboard_bucket, static_prefix)
+    )
+
+
+# pylint: disable=too-many-arguments
+def get_and_insert_static_tables(
+    table_name: str,
+    feed_info_path: str,
+    session: sa.orm.session.sessionmaker,
+    insert_table: str,
+    columns_to_pull: List[str],
+    int64_cols: Optional[List[str]] = None,
+    bool_cols: Optional[List[str]] = None,
+    time_to_seconds_cols: Optional[List[str]] = None,
+) -> None:
```

Comment:
This function is responsible for the common ETL logic for the static schedule tables.